### PR TITLE
Port carrying from Delta-v

### PR DESCRIPTION
**Pull request by Leander-0**
## About the PR
Adds the carrying system and gives the component to all available species. Carry the dead, carry the wounded, carry the people you want to throw, carry the prisoners, carry the felinids and launch them at mach 5 to the void of space where they belong when they spam meows.

## Why / Balance
Can be used by pirates to throw mates for boarding, sec for carrying prisoners, medics for carrying the wounded or just to throw away some people to space.

## Technical details

Most of the carrying code is already in this repository, this just takes the changes done on delta with some little changes so it could build, "it works on my machine" right now.

## Media

https://github.com/new-frontiers-14/frontier-station-14/assets/80334192/20c8f86e-5eac-48df-91f7-01a24eb376b4

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Leander
- add: All species have learned how to carry people and throw them around.

### CODE REVIEW

**Comment by GrandKadoer** [comment]
This is absolutely crucial for EVA rescue operations imo.

File: Content.Server/Resist/EscapeInventorySystem.cs [hunk 1]
```diff
@@ -96,7 +100,14 @@ private void OnEscape(EntityUid uid, CanEscapeInventoryComponent component, Esca
...
 
-        if (TryComp<BeingCarriedComponent>(uid, out var carried))
+        if (TryComp<BeingCarriedComponent>(uid, out var carried)) // Start of carrying system of nyanotrasen.
+        {
+            _carryingSystem.DropCarried(carried.Carrier, uid);
+            return;
+        } // End of carrying system of nyanotrasen.
+
+
+        if (TryComp<BeingCarriedComponent>(uid, out carried))
```
Comment (Cheackraze) [inline Content.Server/Resist/EscapeInventorySystem.cs:110]: youve doubled up this guard clause, we only need one. Preferably the better commented one lol

File: Content.Shared/Nyanotrasen/Carrying/CarryingSlowdownComponent.cs [hunk 2]
```diff
@@ -15,7 +15,7 @@ public sealed partial class CarryingSlowdownComponent : Component
     }
 
     [Serializable, NetSerializable]
-    public sealed partial class CarryingSlowdownComponentState : ComponentState
+    public sealed class CarryingSlowdownComponentState : ComponentState
```
Comment (Cheackraze) [inline Content.Shared/Nyanotrasen/Carrying/CarryingSlowdownComponent.cs:18]: components all need to be partial or they will explode

**Review by Cheackraze — CHANGES_REQUESTED** [review]

**Comment by Leander-0** [comment]
Issues should be fixed

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**